sdk/servicebus/service-bus/package.json

```diff
@@ -75,9 +75,10 @@
     ]
   },
   "dependencies": {
-    "@azure/core-amqp": "1.0.0-preview.3",
+    "@azure/core-amqp": "^1.0.0",
     "@azure/core-http": "^1.0.0",
     "@azure/ms-rest-nodeauth": "^0.9.2",
```

Comment:
`@azure/ms-rest-nodeauth` should no longer be needed now that we are using `@azure/identity`

Reply:
It's still being used in the tests - moved it to dev dependency

Reply:
After the removal of the aadutils file, I believe `@azure/ms-rest-nodeauth` is not used in tests either.